README.md

```diff
@@ -1,12 +1,407 @@
 # xmtp-android
-XMTP client SDK for Android applications written in Kotlin.
 
-**Coming soon!** This is a work in progress and should not be used yet. Follow along in the [tracking issue](https://github.com/xmtp/xmtp-android/issues/1) for updates on progress.
+![Test](https://github.com/xmtp/xmtp-android/actions/workflows/test.yml/badge.svg) ![Lint](https://github.com/xmtp/xmtp-android/actions/workflows/lint.yml/badge.svg) ![Status](https://img.shields.io/badge/Project_Status-Pre_Preview-red)
 
-### Setup
+`xmtp-android` provides a Kotlin implementation of an XMTP message API client for use with Android apps.
 
-```bash
-cp github.properties.example github.properties
+Use `xmtp-android` to build with XMTP to send messages between blockchain accounts, including DMs, notifications, announcements, and more.
+
+This SDK is in **Pre Preview** status and ready for you to experimenting with.
+
+However, we do **not** recommend using Pre Preview software in production apps. Software in this status is likely to change based on feedback.
+
+Specifically, this SDK is currently building out an example app.
+
+Follow along in the [tracking issue](https://github.com/xmtp/xmtp-android/issues/1) for updates.
+
+To learn more about XMTP and get answers to frequently asked questions, see [FAQ about XMTP](https://xmtp.org/docs/dev-concepts/faq).
+
+![x-red-sm](https://user-images.githubusercontent.com/510695/163488403-1fb37e86-c673-4b48-954e-8460ae4d4b05.png)
+
+## Example app
+
+For a basic demonstration of the core concepts and capabilities of the `xmtp-android` client SDK, see the [Example app project](https://github.com/xmtp/xmtp-android/tree/main/example). This is currently a work in progress.
+
+## Install from the Google Maven repository
+
+```gradle
+    implementation 'org.xmtp:android:X.X.X'
```

Comment:
This won't work until https://github.com/xmtp/xmtp-android/pull/32 merges.

Reply:
I've made a pre preview github package here to unblock this https://github.com/xmtp/xmtp-android/packages/1797061